### PR TITLE
(node/foreman.cp.lsst.org) deploy vlan 1700 LSSTCAM dhcp pool

### PR DESCRIPTION
**Pull request by julioconstanzo**
deploy vlan1700 LSSTCAM dhcp pool

### CODE REVIEW

**Review by jhoblitt — CHANGES_REQUESTED** [review]
Please add this subnet and the dhcp pool to ipam.

File: hieradata/node/foreman.cp.lsst.org.yaml
Comment (dtapiacl) [inline hieradata/node/foreman.cp.lsst.org.yaml:1]: LGTM. Im only wondering of the range should be smaller considering they are mostly assigning static ips to their machines now.

**Review by dtapiacl — APPROVED** [review]
LGTM.

**Review by cbarria — COMMENTED** [review]
Now it only has 1 commit and not 4... 👍🏻

**Comment by julioconstanzo** [comment]
both new subnet and vlan were added to IPAM

**Comment by julioconstanzo** [comment]
> Please add this subnet and the dhcp pool to ipam.

done

**Review by jhoblitt — APPROVED** [review]
LGTM. Thank you for updating ipam.

**Pull request merged**